App Frontend/src/views/ManagerAccountView.vue

```diff
@@ -48,7 +39,32 @@ export default {
     }
 	},
 
+  computed: {
+    user () {
+      return this.$store.getters.getUser
+    }
+  },
+
   methods: {
+    async updateWallet(amount) {
+      console.log(this.$store.getters.getUser)
+      try {
+        const response = await fetch(`http://localhost:3000/addWallet/add/${amount}`, {
```

Comment:
For future reference, we can use axios.get() or axios.post() instead of fetch. I find it's a little bit more concise. https://axios-http.com/docs/intro